src/Test.Utilities.Sarif/ResourceExtractor.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities.Sarif
```

Comment:
we'll need to discuss standard namespace conventions for test code at some point, won't randomize this PR with it.

Reply:
Changed to match surrounding classes.